lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -538,6 +538,39 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 	*/
 	setReadOnly( TypedArray.prototype, 'BYTES_PER_ELEMENT', TypedArray.BYTES_PER_ELEMENT );
 
+	/**
+	* Invokes a function once for each array element.
+	*
+	* @name findIndex
+	* @memberof TypedArray.prototype
+	* @type {Function}
+	* @param {Function} callbackFn - function to invoke
+	* @param {*} [thisArg] - function invocation context
+	* @throws {TypeError} `this` must be a typed array instance
+	* @throws {TypeError} first argument must be a function
+	*/
+	setReadOnly( TypedArray.prototype, 'findIndex', function findIndex( fcn, thisArg ) {
+		var result;
+		var buf;
+		var i;
+
+		if ( !isTypedArray( this ) ) {
+			throw new TypeError( format( 'invalid invocation. `this` is not %s %s.', CHAR2ARTICLE[ dtype[0] ], CTOR_NAME ) );
+		}
+		if ( !isFunction( fcn ) ) {
+			throw new TypeError( format( 'invalid argument. First argument must be a function. Value: `%s`.', fcn ) );
+		}
+
+		buf = this._buffer;
+		for (i = 0; i < this._length; i++) {
+			result = fcn.call(thisArg, buf[GETTER](i * BYTES_PER_ELEMENT, this._isLE), i, this);
+			if (result) {
```

Comment:
```suggestion
		for ( i = 0; i < this._length; i++ ) {
			if ( fcn.call( thisArg, buf[ GETTER ] (i * BYTES_PER_ELEMENT, this._isLE ), i, this ) ) {
```

Use consistent spacing.